mappings/net/minecraft/world/gen/chunk/StructureConfig.mapping

```diff
@@ -1,8 +1,29 @@
 CLASS net/minecraft/class_5314 net/minecraft/world/gen/chunk/StructureConfig
+	COMMENT Contains parameters for placement of a single type of {@link net.minecraft.world.gen.feature.StructureFeature} during chunk
+	COMMENT generation.
 	FIELD field_24917 CODEC Lcom/mojang/serialization/Codec;
 	FIELD field_24918 spacing I
+		COMMENT Defines how many chunks make up one cell in the grid used to place structures.
+		COMMENT Each cell in the grid will receive one potential structure start.
+		COMMENT <p>
+		COMMENT Minimum is 1, which means the structure will potentially be placed in every chunk.
+		COMMENT 2 leads to one structure per 2x2 chunks, and so on.
 	FIELD field_24919 separation I
+		COMMENT Defines how varied the placement of a structure within the chunk grid (see spacing) is.
```

Comment:
It might help to define separation along the lines of being the minimum distance in chunks between structures

Reply:
I tried to improve this javadoc and also made a comment about it effectively being a margin that is subtracted from the width/height when actually placing the structure start.